### PR TITLE
GlusterFS upgrade playbook related additional fixes

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>

### CODE REVIEW

File: playbooks/openshift-glusterfs/private/setup_nodes.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@
   - import_role:
       name: openshift_node
       tasks_from: glusterfs.yml
+  - import_role:
+      name: openshift_node
+      tasks_from: storage_plugins/iscsi.yml
```
Comment (vrutkovs) [inline playbooks/openshift-glusterfs/private/setup_nodes.yml]: `ERROR! Could not find specified file in role: tasks/iscsi.yml`

File: roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml [hunk 2]
```diff
@@ -1,4 +1,12 @@
 ---
+- import_tasks: cluster_health.yml
+  vars:
+    l_check_bricks: true
+  when: glusterfs_is_native
+
+- import_tasks: heketi_db_backup.yml
```
Comment (vrutkovs) [inline roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml]: `ERROR! Unable to retrieve file contents
Could not find or access '/home/travis/build/openshift/openshift-ansible/playbooks/openshift-glusterfs/private/heketi_db_backup.yml' on the Ansible Controller.`

Comment (SaravanaStorageNetwork) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml]: Done

File: playbooks/openshift-glusterfs/private/setup_nodes.yml (hunk 1 above)
Comment (SaravanaStorageNetwork) [reply playbooks/openshift-glusterfs/private/setup_nodes.yml]: Done

**Comment by SaravanaStorageNetwork** [comment]
> Looks good, please squash the commits
Done

**Comment by SaravanaStorageNetwork** [comment]
@vrutkovs  @jarrpa  PTAL

**Pull request merged**